zaplings/views.py

```diff
@@ -765,9 +781,9 @@ def record_new_email(request):
 
 def process_rsvp(request):
     record_new_email(request)
-    record_loves(request, isRedirected=False)
```

Comment:
Same as above. Here is the offending commit:
https://github.com/permalife/zaplings-full/commit/8098113f815793ff1a435d3dc45288497818b090#diff-d75b7b863f912cf9e6eb960ca2ab4ad3L760